src/features/goblins/storageHouse/components/StorageItems.tsx

```diff
@@ -41,6 +59,44 @@ export const StorageItems: React.FC = () => {
             );
           })}
         </div>
+
+        <p className="mb-2 underline">Goblin Community Treasury</p>
+        {loading ? (
+          <div className="my-2">Loading...</div>
```

Comment:
For the animated effect of the 3 dots, consider using the classname `loading`
```suggestion
          <div className="my-2 loading">Loading</div>
```

Reply:
TIL 👍